packages/cli-plugin-metro/src/tools/loadMetroConfig.ts

```diff
@@ -69,13 +69,21 @@ export interface ConfigOptionsT {
  * Allows the CLI to override certain defaults in the base `metro.config.js`
  * based on dynamic user options in `ctx`.
  */
-export default function loadMetroConfig(
+export default async function loadMetroConfig(
   ctx: ConfigLoadingContext,
-  options?: ConfigOptionsT,
+  options: ConfigOptionsT = {},
 ): Promise<ConfigT> {
   const overrideConfig = getOverrideConfig(ctx);
-  if (options && options.reporter) {
+  if (options.reporter) {
     overrideConfig.reporter = options.reporter;
   }
+
+  const projectConfig = await resolveConfig(undefined, ctx.root);
```

Comment:
💡 Potentially we could log `projectConfig.filename` every time the CLI runs. This would be useful in the React Native repo where we have multiple Metro configs across test setups — perhaps less useful elsewhere but still informational.

```
Reading Metro config from packages/rn-tester/metro.config.js
```

Thoughts? :)

Reply:
We could put it behind `logger.debug` I guess